### DISCUSSION TITLE
cy.intercept() for all tests

### DISCUSSION THREAD

**Discussion by vicainelli** [cypress-io/cypress#16025] [Questions and Help] [answered]
Hi all, 

I wondering if it's possible to set up some `cy.intercept()` calls for all test cases.
Currently, I just call on `beforeEach` hook, but I need to put the same calls in all files.

Is the same request, for all pages in the application.

**Comment by bahmutov** [comment, marked as answer]
No, you would need to use `beforeEach` to create the `cy.intercept` because the intercepts are automatically reset before every test. You can however place that `beforeEach` hook in your support file https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Support-file then it will run for all tests

**Reply by vicainelli** [reply]
Yeah, worked... thanks :)
